scss/_patterns_pull-quotes.scss

```diff
@@ -29,6 +29,7 @@
     .p-pull-quote__citation {
       @extend %vf-heading-6;
 
+      display: block;
```

Comment:
Would be good to bump version of Vanilla because we are changing SCSS files.

It's not a new feature or bugfix really, but it is a change that when released would need new version. So bump to 3.1.1 please.